independent-projects/resteasy-reactive/server/processor/src/main/java/org/jboss/resteasy/reactive/server/processor/ServerEndpointIndexer.java

```diff
@@ -381,8 +381,9 @@ protected void handleArrayParam(Map<String, String> existingConverters, String e
         builder.setConverter(new ArrayConverter.ArraySupplier(converter, elementType));
     }
 
-    protected void handlePathSegmentParam(ServerIndexedParameter builder) {
```

Comment:
Is this method supposed to be removed?

Reply:
Ouch... No...

Great catch!

Reply:
Fixed, thanks for spotting this dump mistake on my part :)

Reply:
No problem, that's what I am here for 😆